src/Photos.vue

```diff
@@ -22,6 +22,7 @@
 
 <template>
 	<Content app-name="photos">
+		{{MapsInstalled}}
```

Comment:
you cannot use variables that are not in computed/data in the template.
Also please use variables name like `isMapsInstalled` as it's more straightforward :)

Reply:
@jakobroehrl ^

Reply:
will change it, when it's working. 
There is still one error:
Uncaught Error: Could not find initial state undefined of maps
at loadState (index.js:14)
at Module../src/services/MapsInstalled.js (MapsInstalled.js:24)

Reply:
loadstate() require the app name as first argument https://github.com/nextcloud/photos/blob/master/src/services/AllowedMimes.js#L25

Reply:
> 
> 
> loadstate() require the app name as first argument https://github.com/nextcloud/photos/blob/master/src/services/AllowedMimes.js#L25

Found this 2mins before by myself, but it doesn't work neither

Reply:
It works for me :)

Reply:
OK, now it's working. :)